syntax = "proto3";
package LPMsg;

message PathNode
{
    int32   sertype = 1;
    int32   serid = 2;
}

message ReqNoticeServer
{
    int32   serverType = 1;
    repeated  PathNode    path = 2;
}

message AckNoticeServer
{
    int32               state = 1;
    int32               sertype = 2;
    repeated int32      serid = 3;
}